docs/reference/server-config.md

```diff
@@ -359,6 +361,29 @@ Example:
 	</tr>
 </table>
 
+## Hot logging level reload
+We don't support to completely reload the whole configuration yet at present. What we support for now is:
+- increase logging level by signaling `SIGUSR1`
+- decrease logging level by signaling `SIGUSR2`
+
+Example:
+
+To increase logging level
+```
+$ kill -s SIGUSR1 `ps aux | grep "notary-server -config" | grep -v "grep" | awk '{print $2}'`
```

Comment:
I wouldn't be opposed to giving both, so long as we give the caveat that the kill+grep command would only work if you're just running the bare binary.  

It also probably works if you are running this on the same machine as your docker host, and if you are running the container exactly as we give it in docker-compose and not with any kind of wrapper startup script or anything.

But the kill+grep command doesn't work for me for instance because I use either docker for mac or docker machine, so giving the `docker kill` command is important I think.